cli/alsatian-cli-options.ts

```diff
@@ -73,7 +73,7 @@ export class AlsatianCliOptions {
          const argumentIndex = this._getArgumentIndexFromArgumentList(args, "timeout", "t");
 
          // filter out the timeout argument and its value
-         return args.filter((value, index) => {
+         return args.filter((_, index) => {
```

Comment:
I think leave it as it is. Any odd maskings around `value` (such as reassigning it, using an underscore) could cause confusion.